benchmarks/benchmark.py

```diff
@@ -4,11 +4,10 @@
 import spacy
 import stanza
 
-import syntok
 from syntok.tokenizer import Tokenizer
 import syntok.segmenter as syntok_segmenter
 
-from english_golden_rules import GOLDEN_EN_RULES
+from .english_golden_rules import GOLDEN_EN_RULES
```

Comment:
This relative import makes it difficult to run as a script. How are you running this benchmark?

Reply:
I was running it as `python -m benchmarks.benchmark`